pkgs/applications/editors/jupyter/console.nix

```diff
@@ -0,0 +1,43 @@
+# Jupyter console with the given kernel definitions
+
+{ python3
+, jupyter-kernel
+, lib
+}:
+
+let
+  mkConsole = {
+    definitions ? jupyter-kernel.default
+    , kernel ? null
+  }:
+    let
```

Comment:
We should avoid nesting let in if possible

Reply:
Hmm, I could inline it into the result attrset and then use `rec`, but I thought that was being discouraged...

Reply:
let ins themselves are recursive so we could just flatten this

Reply:
I just inlined the inner binding